frontend/src/components/Table/DateCell.tsx

```diff
@@ -0,0 +1,6 @@
+import { ILeaseTerm } from 'interfaces';
+import { CellProps } from 'react-table';
+import { prettyFormatDate } from 'utils';
+
+export const DateCell = ({ cell: { value } }: CellProps<ILeaseTerm, string>) =>
```

Comment:
nit: Generic component should expect to be `CellProps<Date, string>` instead of ILeaseTerm